internal/upgradecheck/http.go

```diff
@@ -152,9 +152,8 @@ func CheckForUpgradesScheduler(channel chan bool,
 	log := logging.FromContext(ctx)
 	defer func() {
 		if err := recover(); err != nil {
-			log.Error(
-				fmt.Errorf("%s", err),
-				"Error in scheduling upgrade checks",
+			log.V(1).Info("encountered panic in upgrade check",
+				"response", err,
```

Comment:
👍🏻 I like that we don't try to figure out what `recover()` returns anymore. As I recall, the logging package will decide how to render this `err` value.